content/blog/dplyr-1-0-4-if-any/index.Rmd

```diff
@@ -0,0 +1,121 @@
+---
+output: hugodown::md_document
+
+slug: dplyr-1-0-4-if-any
+title: dplyr 1.0.4 if_any() and if_all()
+date: 2021-02-01
+author: Romain Francois
+description: >
+    Minor release with major performance improvements for across() and 
+    two new functions if_any() and if_all().
+
+photo:
+  url: https://unsplash.com/photos/nQz49efZEFs
+  author: Mattias Olsson
+
+categories: [package] 
+tags: [dplyr]
+---
+
+We're happy to announce the release of [dplyr](https://dplyr.tidyverse.org) 1.0.4. dplyr is the data
+manipulation package of the [tidyverse](https://www.tidyverse.org). 
+
+This release dramatically improves the performance of `across()` and introduces 
+the new functions `if_all()` and `if_any()` as companions to `filter()`. 
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("dplyr")
+```
+
+This blog post will discuss the two most important changes of the 1.0.4 release: 
+the new `if_any()` and `if_all()` functions, and performance improvements for `across()`.
+
+You can see a full list of changes in the [release notes]({ github_release })
+
+```{r setup}
+library(dplyr, warn.conflicts = FALSE)
+```
+
+## if_any() and if_all()
+
+The new `across()` function introduced as part of dplyr 1.0.0 is starting to be 
+a successful addition to dplyr. In case you missed it, `across()` lets you 
+conveniently express a set of actions to be performed across a tidy selection 
+of columns. 
+
+However useful `across()` may be within `summarise()` and `mutate()`, we discovered
```

Comment:
I've tried to avoid explaining in terms of across() because we want to make across() warn when used in filter() later

Reply:
Ah good point. Maybe something more like:  `across()` is very useful within `summarise()` and `mutate()`, but while it's technically possible to use within `filter()`, it's not at all obvious what it might do.